### ISSUE TITLE
`title` tag is removed from the body 

### ISSUE THREAD

**Issue by stramel** [iconify/tools#27]
The `title` tag is being stripped out during the `importDirectory` call. This prevents someone from being able to disable the plugin `removeTitle` during the `runSVGO` call to keep the `title` tag.

**Comment by cyberalien** [comment]
Added option `keepTitles` to `importDirectory` function in version 3.0.0-beta.4, which you can install using `@iconify/tools@next`.

```js
importDirectory('svg', {keepTitles: true})
```

**Comment by stramel** [comment]
Just tested with the `3.0.0` version and it worked correctly when adding to both `importDirectory` and `cleanupSVG` calls. Thank you!

**Comment by stramel** [comment]
@cyberalien I did notice that the `title` tag is getting wrapped by a `g` tag and isn't getting removed by `removeTitle` plugin in `svgo`. Is there a way to prevent `title` from being wrapped?

**Comment by cyberalien** [comment]
What code are you using that causes content to be wrapped in `<g>`?

**Comment by stramel** [comment]
Sorry, I should have specified in my earlier comment:


Input Code:

```svg
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><g fill="none" stroke="currentColor"><title>TEST</title><path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M12 6V4m0 2a2 2 0 100 4m0-4a2 2 0 110 4m-6 8a2 2 0 100-4m0 4a2 2 0 110-4m0 4v2m0-6V4m6 6v10m6-2a2 2 0 100-4m0 4a2 2 0 110-4m0 4v2m0-6V4"/></g></svg>
```

`svgo` options: 
```js
           {
                multipass: true,
                plugins: [{
                    name: 'preset-default',
                    params: {
                      overrides: {
                        // or disable plugins
                        removeTitle: false,
                      },
                    },
                  }]
            }
```

`svgo` output:
```svg
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><g fill="none" stroke="currentColor"><title>TEST</title><path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M12 6V4m0 2a2 2 0 100 4m0-4a2 2 0 110 4m-6 8a2 2 0 100-4m0 4a2 2 0 110-4m0 4v2m0-6V4m6 6v10m6-2a2 2 0 100-4m0 4a2 2 0 110-4m0 4v2m0-6V4"/></g></svg>
```

**Comment by cyberalien** [comment]
I think you are confusing different code. Iconify Tools is not a wrapper for SVGO, clean up function doesn't use SVGO at all.

**Comment by cyberalien** [comment]
Fixed in `3.0.1`
